src/messages_zh_cn.properties

```diff
@@ -1,110 +1,159 @@
+error.PLAYER_NOT_FOUND=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u73a9\u5bb6\u4e0d\u5b58\u5728\u6216\u672a\u5728\u7ebf\u002e
+error.ISLAND_NO_OWNER=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u8981\u8c03\u6574\u4e3a\u975e\u516c\u5f00\u5bb6\u56ed\u002c\u0020\u8bf7\u5148\u8bbe\u7f6e\u5c9b\u4e3b\u002e
+error.NO_BIOME_FOUND=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u60a8\u8f93\u5165\u7684\u751f\u7269\u7fa4\u7cfb\u4e0d\u5b58\u5728\u002e
+error.NOT_ON_ISLAND=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u4f60\u4e0d\u5728\u4e00\u4e2a\u5bb6\u56ed\u5c9b\u5185\u002e
+error.NO_LOCATIONS_FOR_BIOME=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u8be5\u751f\u7269\u7fa4\u7cfb\u65e0\u53ef\u7528\u533a\u57df\u002e
+error.ISLAND_NOT_FOUND=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u5c9b\u5c7f\u4e0d\u5b58\u5728\u002e
+error.HOME_NOT_FOUND=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u5bb6\u56ed\u4e0d\u5b58\u5728\u0020\u003a\u0028
+error.NOT_TRUSTED=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u4f60\u6ca1\u6709\u6743\u9650\u8fd9\u4e48\u505a\u002e
+error.INVALID_ISLAND_SIZE=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u5bb6\u56ed\u5c9b\u5927\u5c0f\u8d85\u51fa\u9650\u5236\u002e
+error.NO_WORLDEDIT=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u672a\u68c0\u6d4b\u5230\u0020\u0057\u006f\u0072\u006c\u0064\u0045\u0064\u0069\u0074\u002e
+error.ERROR=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u5185\u90e8\u9519\u8bef\u002e\u0020\u8bf7\u8054\u7cfb\u7ba1\u7406\u5458\u002e
+error.COOLDOWN=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u4f60\u521a\u521a\u906d\u5230\u4e86\u4f24\u5bb3\u002e\u0020\u4f20\u9001\u5c06\u5728\u0020 {0}\u79d2\u0020\u540e\u5f00\u59cb\u002e
+error.ISLAND_SAVE_ERROR=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u65e0\u6cd5\u4fdd\u5b58 {0}.schem.
+error.INSUFFICIENT_FUNDS=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u4f60\u6ca1\u6709\u8db3\u591f\u7684\u94b1\u002e
+error.ISLAND_UNFINISHED=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u60a8\u7684\u5bb6\u56ed\u5c9b\u4ecd\u5728\u751f\u6210\u002e
 error.PRIVATE_ISLAND=info.CLEAR_CONFIRM
```

Comment:
This line is supposed to say `error.PRIVATE_ISLAND=\u00a77[\u00a74\u00a7lError\u00a77] \u00a7cYou can't check private island's info.` in English